src/Symfony/Component/Console/Application.php

```diff
@@ -594,6 +594,8 @@ public function all($namespace = null)
      */
     public static function getAbbreviations($names)
     {
+        @trigger_error(sprintf('%s is deprecated as of 3.2 and will be removed in 4.0.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
@nicolas-grekas While looking for code not covered I noticed that this method wasn't used anywhere so I thought that it should be removed. What's your opinion ?